python/cugraph-dgl/cugraph_dgl/dataloading/dataset.py

```diff
@@ -133,7 +133,7 @@ def set_input_directory(self, input_directory):
         self._sampled_files = os.listdir(input_directory)
         self._batch_to_fn_d = {
             i: get_batch_fn_batch_start(i, self._sampled_files)
-            for i in range(0, self.num_batches + 1)
+            for i in range(0, self.num_batches)
```

Comment:
Unrelated to PR, Caught likely now after updating https://github.com/rapidsai/cugraph/pull/3249